stubs/channels/channels/testing/application.pyi

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+from asgiref.testing import ApplicationCommunicator as BaseApplicationCommunicator
+
+def no_op() -> None: ...
+
+class ApplicationCommunicator(BaseApplicationCommunicator):
+    async def send_input(self, message: Any) -> None: ...
+    async def receive_output(self, timeout: float = 1) -> Any: ...
```

Comment:
I have no idea whether the comment below is accurate, but we need a comment explaining the use of `Any`.

```suggestion
    # The type of the sent or received message is application-specific.
    async def send_input(self, message: Any) -> None: ...
    async def receive_output(self, timeout: float = 1) -> Any: ...
```